tests/installation/bootloader_ofw.pm

```diff
@@ -23,14 +23,11 @@ use utils;
 # hint: press shift-f10 trice for highest debug level
 sub run() {
     assert_screen "bootloader", 15;
-    if (get_var("UPGRADE")) {
-        send_key_until_needlematch 'inst-onupgrade', 'up';
-    }
-    elsif (get_var("ZDUP") || get_var("ONLINE_MIGRATION")) {
+    if (get_var("ZDUP") || get_var("ONLINE_MIGRATION")) || get_var("UPGRADE") && get_var("PATCH") {
```

Comment:
Don't we need a version of if get_var("UPGRADE") && !get_var("PATCH") to preserve the old behaviour for send_key_until_needlematch to inst-onupgrade?